transform/mattermost-analytics/models/marts/product/nps/_nps__models.yml

```diff
@@ -37,3 +37,87 @@ models:
       - name: user_email
         description: The email of the user who submitted feedback (optional).
         tags: [pii]
+
+  - name: fct_nps_score
+    description: |
+      This model contains NPS user count metrics on server parameters (server_id, server_version, user_role) sent from Rudderstack and Segment sources.
+      Score is calculated for servers after April 2020, since some servers from Segment reported the user_id's as empty prior.
+      This raises divide by zero error with the NPS score calculation.
+
+    columns:
+      - name: activity_date
+        description: The date the NPS score event was sent. 
+        tests:
+          - not_null
+      - name: server_id
+        description: The ID of the server the NPS score event originated from..
+        tests:
+          - not_null
+      - name: server_version
+        description: The server's version.     
+      - name: user_role
+        description: The role of the user who submitted feedback.        
```

Comment:
```suggestion
```
`user_role` column doesn't exist in table. Seems like a leftover from previous commits.

Reply:
Dropped this.